daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/RuntimeData.scala

```diff
@@ -99,31 +98,48 @@ object TermRuntimeData {
 
 }
 
+/**
+ * Base runtime data structure for all terms
+ *
+ * These transient Delay-type args are part of how we
```

Comment:
Is it worth removing "transient" in this comment. There are no longer any TransientParam stuff.